### PR TITLE
Add Kubernetes deployment

### PR DESCRIPTION
**Pull request by davidgasquez**

### CODE REVIEW

File: crawler/crawler.py [hunk 1]
```diff
@@ -113,32 +112,28 @@ def stripe_events_for_range(start_date, end_date, filter):
...
-    finally:
-        p.close()
+    return KinesisProducer(KINESIS_STREAM_NAME,
+                           batch_size=BATCH_SIZE,
+                           max_retries=KINER_MAX_RETRIES,
+                           threads=KINER_THREADS)
 
 def process_events(events, transformer, put_record):
     transformed_events = transformer(events)
-    serialized_events = [e.SerializeToString() for e in transformed_events]
```
Comment (michael-erasmus) [inline crawler/crawler.py]: Curious why we're rolling back this change?

File: crawler/crawler.py [hunk 2]
```diff
@@ -113,32 +112,28 @@ def stripe_events_for_range(start_date, end_date, filter):
...
 def make_producer():
-    try:
-        p = KinesisProducer(KINESIS_STREAM_NAME,
-                            batch_size=BATCH_SIZE,
-                            max_retries=KINER_MAX_RETRIES,
-                            threads=KINER_THREADS)
-        yield p
-    finally:
-        p.close()
+    return KinesisProducer(KINESIS_STREAM_NAME,
```
Comment (michael-erasmus) [inline crawler/crawler.py]: I would love to perhaps keep this change too, at least until we have a fix for the issue where the Kiner process hangs around if you don't call `close` on it

Comment (davidgasquez) [reply crawler/crawler.py]: Hey @michael-erasmus! After some tests, I found that the issue was related to Docker not closing the container properly when receiving the `CTRL + C` keys. As a workaround, I've found closing the container with `docker stop` or `docker kill` works. 

I shouldn't have committed this as I was only trying to reproduce the issue!

File: crawler/crawler.py (hunk 1 above)
Comment (davidgasquez) [reply crawler/crawler.py]: I changed this to be able to print events locally and also forgot to revert the changes 🤦‍♂️

Comment (davidgasquez) [reply crawler/crawler.py]: Just pushed a commit that reverts these changes! Thanks for the review @michael-erasmus!

Comment (michael-erasmus) [reply crawler/crawler.py]: Thanks!

**Pull request merged**